concepticondata/conceptlists/Verheyen-2019-1000.tsv

```diff
@@ -0,0 +1,1001 @@
+Verheyen-2019-1000-434	434	koppig	stubborn	STUBBORN	3634	6.45	5.5	3.95	5.75	5.9	4.6	3.3	6	2	18713	6	661	611	2.88369026871884	0.000514522049755
+Verheyen-2019-1000-435	435	korrelig	granular			8.95	3.84210526315789	4.95	3.7	2.25	5.9	2.8	8	3	65843	1	68	10	0.246020110286115	6.75600829478103E-05
+Verheyen-2019-1000-436	436	kort	short	SHORT	1645	4.1	3.55	4.4	3.95	5.95	5.45	3.75	4	1	17474	14	14756	4000	7.12125937582388	0.001221187305408
+Verheyen-2019-1000-437	437	kostbaar	rich	RICH	712	8.35	4.85	4.75	5.45	3.85	4.9	5.4	8	2	55886	1	1325	606	0.816454145593196	8.08664352031736E-05
+Verheyen-2019-1000-438	438	kostelijk	precious	PRECIOUS	3002	8.8	4.4	4.4	5.15	2.6	4.1	3.85	9	3	89142	2	259	57	0.503703930405361	9.70039262362737E-05
+Verheyen-2019-1000-439	439	koud	cold	COLD	1287	4.15	2.75	4.25	3.7	5.7	5.65	2.9	4	1	8281	6	5789	5168	27.2792030264458	0.00229926759633
+Verheyen-2019-1000-440	440	krachtig	forceful			7.35	6.2	5.2	5.8	5	4.75	5.5	8	2	55611	2	2778	925	1.49185882713605	0.000291958745817
+Verheyen-2019-1000-441	441	krap	scant			9.47368421052632	3.3	4.2	3.55	2.95	4.45	2.65	4	1	15588	13	205	330	0.304408735845921	5.54610324011142E-05
+Verheyen-2019-1000-442	442	kras	youthful			7.52631578947368	3.68421052631579	4.41176470588235	3.76470588235294	2.35	3.63157894736842	2.68421052631579	4	1	17025	13	126	8	0.245119414696251	8.15140175212996E-05
+Verheyen-2019-1000-443	443	kritiek	critical			9.35	4.4	5.35	5.2	4.75	4	3.05	7	2	57030	0	204	129	1.53339266080504	0.000334555994783
+Verheyen-2019-1000-444	444	kritisch	critical			11.35	5.35	4.25	5.35	5.2	4.65	3.35	8	2	65861	0	1961	141	0.641703166936328	0.000164027069071
+Verheyen-2019-1000-445	445	krom	bent	CROOKED	297	4.675	3.5	5.3	3.1	3.7	6	3.15	4	1	15745	11	394	170	2.2246688081325	0.000474853565042
+Verheyen-2019-1000-446	446	kruidig	spicy	SPICY	544	8	4.35	4.65	5	3.2	4.55	4.4	7	2	31452	1	145	19	0.197029354331083	5.15104548422952E-05
+Verheyen-2019-1000-447	447	kuis	chaste			10.55	3.3	4.5	3.35	2.05	3.7	3.5	4	1	16932	12	191	24	0.334466093562832	7.89985344463679E-05
+Verheyen-2019-1000-448	448	kunstzinnig	artistic			10.8	4.4	4.4	5	3	3.9	4.35	11	3	64035	0	191	33	0.218324737488974	4.05894183617378E-05
+Verheyen-2019-1000-449	449	kwaad	evil	EVIL	45	4.35	5.4	5.2	6.15	5.65	6.1	2.15	5	1	23224	1	3609	7307	10.2869693108768	0.001110864010186
+Verheyen-2019-1000-450	450	kwalitatief	qualitative			12.4	4.35	4.2	5.25	4.7	2.7	4.35	11	4	76082	0	433	6	0.057507545979803	1.39994931285333E-05
+Verheyen-2019-1000-451	451	kwaliteitsvol	high-quality			11	4.3	3.65	5.35	3.2	3.7	5.25	13	4	78699	0	0	1	0.075729851123961	7.99227222772613E-06
+Verheyen-2019-1000-452	452	kwijt	rid			4	3	5.4	3.15	5.45	3.75	2.2	5	1	15462	3	2296	6365	2.19593159800405	0.000300312232303
+Verheyen-2019-1000-453	453	laag	low	LOW	721	4.55	2.85	4.75	2.55	6.05	5.9	2.95	4	1	28122	13	7434	2580	3.32472538438198	0.000973205581938
+Verheyen-2019-1000-454	454	laat	late	BE LATE	425	5.45	3.45	3.84210526315789	3.1	6.1	5.55	3.15	4	1	32945	11	31168	14337	2.06491747460054	0.0004839
```

Comment:
just like in English, lang can be either referring to time periods or spatial relations